MaterialLib/PorousMedium/Permeability/Permeability.h

```diff
@@ -68,7 +69,7 @@ class Permeability
         return _intrinsic_permeability_tensor;
     }
 
-private:
+protected:
     ParameterLib::Parameter<double> const& _permeability_parameter;
     int const _dimension;
     mutable Eigen::MatrixXd _intrinsic_permeability_tensor;
```

Comment:
Try to remove this member. I don't see where it is used. In two `getValue` functions the tensor is computed every time.